recipes/ipapy2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ipaPy2" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8a94928064ea6a97f5b8800f2ff81faaa2b1417f5765a22755a9543102b9cc4e
+
+build:
+  noarch: generic
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - molmass ==2021.6.18
+    - pandas
+    - pip
+    - python
+    - scipy ==1.8.1
+    - tqdm ==4.64.0
+  run:
+    - molmass ==2021.6.18
+    - pandas
+    - python
+    - scipy ==1.8.1
+    - tqdm ==4.64.0
```

Comment:
please try to avoid strict pinnings if possible - are these really necessary? Please try using more generic lower pins

Reply:
I didn't test the package with more generic lower pins. I kept them this strict to make sure that updates on the requirements will not break the package in the future

Reply:
This is for sure true, but this makes resolving the dependencies a lot more complicated. Also, there are some internal pinnings by bioconda which might conflict with some of the strict pinnings. I would start with specifying these current versions as the lower bounds.